docs/xks/operator-guide/kubernetes/eks.md

```diff
@@ -101,6 +101,133 @@ Run Terraform in the following order:
 Make sure that you only have one peering request open at the same time, else the accepter side won't be able to find a unique request.
 Now you should be able to see the VPC peering connected on both sides.
 
+## Update cluster version
+
+Updating EKS cluster version can not be done by updating Terraform code only, it also involves the AWS CLI and kubectl. Find your EKS version to upgrade to here: [EKS versions](https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html)
+
+For further information on the AWS CLI commands used in this section, please refer to [AWS EKS CLI](https://docs.aws.amazon.com/cli/latest/reference/eks/index.html)
+
+### Update the control plane
+
+Get the name of the cluster to update:
+
+```bash
+aws eks list-clusters --region eu-west-1
+```
+
+Update the control plane version by running the following command with the actual values for \<cluster-name\> and \<version>\:
+
+```bash
+aws eks update-cluster-version --region eu-west-1 --name <cluster-name> --kubernetes-version <version>
+```
+
+The above command provides an id that can be use to check the status of the update:
+
+```bash
+aws eks describe-update --region eu-west-1 --name <cluster-name> --update-id <id>
+```
+
+The update is finished when status is `Successful`. Previous updates have approximately taken 45 minutes.
+
+In the `aws-eks/variables/<environment>.tfvars` Terraform file that corresponds to the actual environment, update the `kubernetes_version` in `eks_config` and make a `terraform plan`. No difference in the plan output is expected. Also make a `terraform apply` just to make sure state the state is updated (might not be needed).
+
+### Update the addons
+
+The addons to be updated are `coredns` and `kube-proxy`. Can be verified with:
+
+```bash
+aws eks list-addons --region eu-west-1 --cluster-name <cluster-name>
+```
+
+The current addon version can be found by:
+
+```bash
+aws eks describe-addon --region eu-west-1  --cluster-name <cluster-name>  --addon-name <addon-name>
+```
+
+The version to update to can be found by:
+
+```bash
+aws eks describe-addon-versions --region eu-west-1 --kubernetes-version <version> --addon-name <addon-name>
+```
+
+The addons is updated with:
+
+```bash
+aws eks update-addon --region eu-west-1 --cluster-name <cluster-name> --addon-name <name> --addon-version <version> --resolve-conflicts OVERWRITE
+```
+
+Check that the status of new addon vestion is `ACTIVE`:
+
+```bash
+aws eks describe-addon --region eu-west-1  --cluster-name <cluster-name>  --addon-name  <name>
+```
+
+The corresponding Pods can be checked by:
+
+```bash
+kubectl get pods -n kube-system
+```
+
```

Comment:
Mention that it worth running a general health check.
Maybe curl: https://ingress-healthz.qa.customer.se/